linkis-computation-governance/linkis-engineconn/linkis-engineconn-plugin-core/src/main/scala/org/apache/linkis/manager/engineplugin/common/conf/EnvConfiguration.scala

```diff
@@ -41,13 +41,15 @@ object EnvConfiguration {
   val ENGINE_CONN_CLASSPATH_FILES =
     CommonVars("wds.linkis.engineConn.files", "", "engineConn额外的配置文件")
 
-  val metaspaceSize = if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8)) {
-    "-XX:MaxMetaspaceSize=256m -XX:MetaspaceSize=128m"
+  val MAX_METASPACE_SIZE = CommonVars("linkis.engineconn.metaspace.size.max", "256m")
+
+  lazy val metaspaceSize = if (SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8)) {
```

Comment:
maybe the judge can be deleted, the min jdk is required 1.8

Reply:
yes